packages/common/src/ConfigResponse.ts

```diff
@@ -105,6 +106,16 @@ export interface GSNConfig {
    */
   paymasterAddress?: Address
 
+  /**
+  * The address of the token paymaster contract used by TokenPaymasterProvider
+  */
+  tokenPaymasterAddress?: Address
+
+  /**
+  * Fields required by TokenPaymasterProvider for the supported tokens
+  */
+  tokenPaymasterDomainSeparators?: Record<Address, EIP712Domain>
```

Comment:
Does not have to be `?` as the `Record<>` is allowed to be empty, I believe.

Reply:
Fixed